front-end/packages/help/getting-started.md

```diff
@@ -1,26 +1,114 @@
+Language section shows which languages are supported in your installation. Language checkboxes select languages for text generation.
 
-Vestibulum vitae odio vel odio gravida tincidunt. Phasellus vehicula, ex sed vehicula semper, arcu turpis tincidunt lacus, quis egestas augue lectus non tellus. Maecenas est nunc, semper eget tellus nec, congue dictum libero. Nunc sit amet tellus maximus, mattis est eget, scelerisque felis. Mauris dapibus eros nec ipsum venenatis facilisis. Maecenas interdum auctor neque, accumsan dapibus erat. Vivamus vitae ullamcorper quam. Nulla blandit sapien id eleifend blandit. Curabitur id volutpat enim. Nunc tellus mauris, blandit nec fringilla vitae, sollicitudin in ipsum. Cras viverra sapien felis, et lobortis diam luctus at. Quisque et dui id turpis finibus accumsan sed eu dolor.
+![data](help/language.png)
 
-Aliquam maximus augue a augue auctor tempor. Suspendisse sit amet dolor consectetur, bibendum orci sit amet, ornare elit. Etiam mauris diam, iaculis eu nunc non, mattis molestie nunc. Vivamus tempus volutpat augue at aliquet. Suspendisse potenti. Pellentesque vestibulum commodo tincidunt. Ut vel est suscipit ex dignissim imperdiet. Vivamus rhoncus eros metus, ut ornare arcu semper a. Morbi id pretium turpis. Sed sed nisi sed tortor placerat pulvinar. Quisque cursus felis ex, et vestibulum lacus sodales a. Nunc non finibus metus. Ut gravida, enim sed laoreet commodo, nisl turpis lobortis ante, iaculis dictum sapien nibh eget nibh. Phasellus accumsan tempor mi a gravida. Vivamus imperdiet dictum nibh, nec pulvinar mi rhoncus in. Maecenas molestie posuere blandit.
+### Text Generation Results
 
+This section will show the resulting texts. Each block of text represents variations of the text given the same document plan and current row of data. A flag next to text shows the language of the text.
+
+![results](help/results.png)
+
+In cases when no text can be generated (due to missconstructed document plan), you will see 'No Variants.' message.
+
+### AMR
+
+This is a core element of the Accelerated Text engine. AMR - Abstract Meaning Representations - define what kind of messages can be constructed about the data. Choose from the AMR list bellow what you want to communicate to your users. AMR must be dragged to the Document Plan canvas to the left (Document Plan construction elements are described in the section bellow).
```

Comment:
"a core element" -> "the core element"; "AMR -" -> "AMRs -"; "AMR must" -> "AMR block must"